### DISCUSSION TITLE
Light variant proposal

### DISCUSSION THREAD

**Discussion by zoedsoupe** [daltonmenezes/aura-theme#198] [Ideas]
There's any idea or roadmap to make a light variant of this theme? I know most people prefer to use always a dark theme, however there are some cases where a light theme can really help, like:

- on daytime, a dark theme is harder to see
- on daytime, a light theme has less overload to our eyes

also, there are some studies to understand the importance of light VS dark themes
- https://www.nngroup.com/articles/dark-mode/

so i always ask for dark theme creators about a proposal to make a light variant, that's because I have medical reasons to use light themes on daytime and use dark theme for the night.

is it seems possible? also, awesome and beautiful theme! great work!

**Comment by daltonmenezes** [comment]
Thanks @zoedsoupe 💜

This is related to #33
